plugins/services/src/js/constants/ServiceActionItem.js

```diff
@@ -1,8 +1,10 @@
+import UserActions from '../../../../../src/js/constants/UserActions';
+
 const ServiceActionItem = {
   CREATE: 'create',
   CREATE_GROUP: 'create_group',
   EDIT: 'edit',
-  DESTROY: 'destroy',
+  DESTROY: UserActions.DELETE,
```

Comment:
Let's weed out `DESTROY` word completely. Feels a bit awkward to see `DESTROY: 'delete'` here. Otherwise it will inevitably create a confusion on a later stage.